README.md

```diff
@@ -75,11 +75,32 @@ The queue is an `array` containing Brightcove video ID strings.
 ```
 $ npm test
 ```
-(Requires Firefox)
+Requires Firefox (v38.0.0 to test with polyfills and mirror CircleCI)
 
 
 ## Migration Guide
 
+Migrating from 2.0 to 3.0
```

Comment:
Previously o-video could also be set up with a data object matching that fetched from next-video/api - worth explicitly mentioning that this should still work if it's set up with a different data object matching that fetched from next-media-api?